packages/lesswrong/server/notificationCallbacks.tsx

```diff
@@ -136,8 +136,8 @@ getCollectionHooks("Posts").updateAsync.add(async function eventUpdatedNotificat
     (
       !_.isEqual(newPost.mongoLocation, oldPost.mongoLocation) ||
       (newPost.joinEventLink !== oldPost.joinEventLink) ||
-      (newPost.startTime !== oldPost.startTime) || 
-      (newPost.endTime !== oldPost.endTime)
+      !moment(newPost.startTime).isSame(moment(oldPost.startTime)) ||
+      !moment(newPost.endTime).isSame(moment(oldPost.endTime))
```

Comment:
These were always true, now they actually compare the times

Reply:
😢